packages/ember-htmlbars/tests/helpers/bind_attr_test.js

```diff
@@ -599,3 +599,25 @@ QUnit.test("src attribute bound to null is not present", function() {
 
   ok(!view.element.hasAttribute('src'), "src attribute not present");
 });
+
+QUnit.test('specifying `<div {{bind-attr style=userValue}}></div>` is [DEPRECATED]', function() {
+  view = EmberView.create({
+    userValue: '42',
+    template: compile('<div {{bind-attr style=view.userValue}}></div>')
+  });
+
+  expectDeprecation(function() {
+    runAppend(view);
+  }, /Allowing dynamic content in a `style` attribute is deprected, please preform a security audit and once verified use `{{{property}}}` instead of `{{property}}`./);
```

Comment:
Should this maybe not deprecate, since `{{bind-attr}}` itself will soon be deprecated? Seems more confusing that useful.